src/app.scss

```diff
@@ -11,60 +11,57 @@
 
 // Passthrough for layout and styling purposes, to enable main page parts to participate in the grid
 .pf-v5-c-page__main-section,
-.pf-v5-c-sidebar,
-.pf-v5-c-sidebar__main,
-.pf-v5-c-sidebar__content {
+.files-view-stack {
     display: contents;
 }
 
-// Move the scrollbar to the content area.
-// Do this by triggering the overflow on the fileview wrapper and sidebar panel
-// when content doesn't fit. (Parent card is also needed to overflow, else the
-// children won't here in this context.)
-.pf-v5-c-sidebar__content > .pf-v5-c-card,
-.fileview-wrapper,
-.sidebar-panel {
-    overflow: auto;
-}
-
 // Content area should stretch, so context menu works even when the content doesn't fill the area
+.files-view-stack .pf-v5-c-card,
 .fileview-wrapper {
     block-size: 100%;
+    overflow: auto;
 }
 
 .pf-v5-c-page__main {
     gap: var(--pf-v5-global--spacer--md);
     display: grid;
     grid-template:
-        "topbar topbar topbar topbar" auto
-        ". content sidebar ." 1fr
-        ". . . ." 0
-        / 0 1fr 18rem 0;
+        "topbar topbar topbar" auto
+        ". content ." 1fr
+        ". files-footer-info ." auto
+        ". . ." 0
+        / 0 1fr 0;
     inline-size: 100vw;
     block-size: 100vh;
 }
 
-.pf-v5-c-sidebar__content > .pf-v5-c-card {
+.files-empty-state,
+.files-view-stack > .pf-v5-c-card {
     grid-area: content;
 }
 
-.sidebar-panel {
-    grid-area: sidebar;
+.files-view-stack > .files-footer-info {
+    margin-block-start: 0;
+    grid-area: files-footer-info;
 }
 
 @container (width < 768px) {
     // Change the general layout in smaller screen sizes
     .pf-v5-c-page__main {
+        gap: var(--pf-v5-global--spacer--sm);
         grid-template:
             "topbar" auto
             "." var(--pf-v5-global--spacer--md)
             "content" 1fr
-            "sidebar" 8rem
+            "files-footer-info" auto
             / 1fr;
         row-gap: 0;
     }
```

Comment:
We could redo the grid layout with something simpler, using named tracks (starting and stopping lines on the endges to stretch to) instead of named spaces (the name of the content area), like this:

```css
.container {
  --gutter: 1rem;
  display: grid;
  grid-template: auto 1fr auto 0 / [full-start] var(--gutter) [content] 1fr var(--gutter) [full-end];
  gap: 0.5rem;
}

.toolbar {
  grid-column: full;
}

.content,
.footer {
  grid-column: content;
}
```

(PF spacers instead of rems, of course. And we could replace var(--gutter) with a PF spacer var too.)

The important things:

1. We don't need a separate layout for smaller widths, as desktop wide is the same layout as even the smallest mobile.
2. Horizontal is specified as full (all the way across, with a start and end) and content (only in the middle (not specifying start and end means it's just the 1 block instead).
3. This doesn't specify the vertical; that just falls into place based on in-document order.
4. The 0 is to use the gap at the end, similar to the `.` placeholder.

To make it more clear, we could also split the shorthand to :

```css
grid-template-columns: [full-start] var(--gutter) [content] 1fr var(--gutter) [full-end];
grid-template-rows: auto 1fr auto 0;
```

```css
grid-template-columns: [full-start] var(--gutter) [content-start] 1fr [content-end] var(--gutter) [full-end];
```

So, the above changes (exact same CSS layout, but perhaps more understandable) would then be:

```css
.container {
  --gutter: 1rem;
  display: grid;
  grid-template-rows: auto 1fr auto 0;
  grid-template-columns: [full-start] var(--gutter) [content-start] 1fr [content-end] var(--gutter) [full-end];
  gap: 0.5rem;
}

.toolbar {
  grid-column: full;
}

.content,
.footer {
  grid-column: content;
}
```